swagger_to_sdk_config_track2.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Can we move this from the root of the repo to under "eng"?

Reply:
Also it would be nice to not use "track2" as the name of the configuration as that is more of a code name.  Given we are putting it in a unique path lets just call it "swagger_to_sdk_config.json"